test/unit_test/passes/onnx/test_gptq.py

```diff
@@ -0,0 +1,75 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from pathlib import Path
+
+import pytest
+import torch
+
+from olive.hardware.accelerator import AcceleratorSpec, Device
+from olive.model.handler.pytorch import PyTorchModelHandler
+from olive.passes.olive_pass import create_pass_from_dict
+from olive.passes.onnx import GptqQuantizer
+
+
+@pytest.mark.skipif(
+    not torch.cuda.is_available(),
+    reason="gptq requires GPU.",
+)
+def test_gptq_default(tmp_path: Path):
+    # setup
+    input_model = PyTorchModelHandler(
+        hf_config={
+            "model_class": "OPTForCausalLM",
+            "model_name": "facebook/opt-125m",
```

Comment:
do you have an estimate for how long this test would take to run on this model? 
Could you try a smaller test model like `"hf-internal-testing/tiny-random-OPTForCausalLM"` like in https://github.com/microsoft/Olive/blob/9d1add1b94304c41ea3fdb0e118cee726c0c5819/test/unit_test/passes/onnx/test_optimum_conversion.py#L49

Reply:
It takes about 200s for two tests on v100, I tried "hf-internal-testing/tiny-random-OPTForCausalLM" model but encounter errors may caused by too small size for int4 weight packing.

Reply:
i see, I was wondering if the small model would be compatible or not. 
our gpu pool is v100 so 3 minutes is not too bad. 

Is there a min limit for the dataset size? Reducing this probably won't help much since the default dataset is still full length?

Reply:
Yeah, small models may not be compatible for some current implementation limitations when do 4bit quantization, and are not needed to be tested? Cause we are focusing on llm and CUDA for now, and I think 125m is really small in such a big model time. 🤣

The dataset could be smaller to just 1 sample, the test time only changes a few seconds.